apps/emqx_bridge_sqlserver/src/emqx_bridge_sqlserver_connector.erl

```diff
@@ -199,7 +199,7 @@ on_start(
     Options = [
         {server, to_bin(Server)},
         {username, Username},
-        {password, emqx_secret:wrap(maps:get(password, Config, ""))},
+        {password, maps:get(password, Config, "")},
```

Comment:
```suggestion
        {password, maps:get(password, Config, emqx_secret:wrap(""))},
```
?

Reply:
Kinda saw no point in hiding default empty string behind a secret. 😅 But I will commit your suggestions anyway if it raises some questions.

Reply:
Sure, but no one needs to know it's an empty string.  :wink: